driver/razerkbd_driver.h

```diff
@@ -110,6 +110,10 @@
 #define USB_DEVICE_ID_RAZER_ORNATA_CHROMA 0x021e
 #endif
 
+#ifndef USB_DEVICE_ID_RAZER_CYNOSA_CHROMA
+#define USB_DEVICE_ID_RAZER_CYNOSA_CHROMA 0x022a
+#endif
```

Comment:
please put this block where it fits sorted by the number (so after USB_DEVICE_ID_RAZER_BLADE_LATE_2016)

Reply:
OK - i hope the next days. Really busy and i don't have always access to keyboard.

Reply:
Could we add the razer cynosa chroma pro with 0x022c?

Reply:
@Vehyla No, the devices are not the same.

Reply:
finally moved (characters are mixed lower- and uppercase - currently lower)